src/components/Carousel.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement } from "react";
+
+type CarouselProps = {
+  childrens: ReactElement[];
+};
+
+function Carousel({ childrens }: CarouselProps) {
+  return (
+    <div id="carouselExampleIndicators" className="carousel slide" data-bs-ride="true">
+      <div className="carousel-indicators">
+        <button
+          type="button"
+          data-bs-target="#carouselExampleIndicators"
+          data-bs-slide-to="0"
+          className="active"
+          aria-current="true"
+          aria-label="Slide 1"
+        ></button>
+        <button
+          type="button"
+          data-bs-target="#carouselExampleIndicators"
+          data-bs-slide-to="1"
+          aria-label="Slide 2"
+        ></button>
+        <button
```

Comment:
Obtenir les boutons aussi via itérateur : leur nombre doit être le même que celui des `children`